torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -310,6 +310,7 @@ def collect_fw_donated_buffer_idxs(
         if (
             t is not None
             and not is_sparse_any(t)
+            and isinstance(t, torch.Tensor)
```

Comment:
Don't know why I need this yet. Going to dig into this more, but cc @bdhirsh in case you have any ideas.